### DISCUSSION TITLE
How to  to customize notification pop up?

### DISCUSSION THREAD

**Discussion by Nishant763** [microsoft/vscode-discussions#568] [Extension Development QnA] [answered]
![image](https://user-images.githubusercontent.com/49840658/232454740-1766309c-40ad-42cf-91e6-21e7cce7e9ff.png)

**Comment by alefragnani** [comment, marked as answer]
Hi @Nishant763 ,

Could you provide more details, or even samples, about what do you want to _customize_ on this pop up (notification)? 

The Notifications API (https://code.visualstudio.com/api/ux-guidelines/notifications) provides a few options, and the samples repo describes a few of them https://github.com/microsoft/vscode-extension-samples/tree/main/notifications-sample. One limitation I remember is about not being able to have _formatting_ (like Markdown). So, if this is one of the customizations you are referring to, this is not currently possible.

Hope this helps
